binary_search/binary_search.py

```diff
@@ -0,0 +1,16 @@
+def binary_search(list, item):
+    low = 0
+    high = len(list) - 1
+
+    while low <= high:
+        mid = low + high
```

Comment:
It isn't binary searching, it is searching from last element to first element) O(n) = n

Reply:
@codewylle fixed